features/calculation_weekly_frequency.feature

```diff
@@ -0,0 +1,26 @@
+Feature:
+  As a enrolee into a Workplace Pension,
+  I would like to calculate pension contribution on my weekly salary.
+
+  Background:
+    Given I am on the YourDetailsPage
+
+    Scenario Outline:
+      Given I enter my age as "<age>"
```

Comment:
This is a really verbose set of steps. I wonder whether we can squash some of these together somehow, whilst still using a table...?
```
Given I enter my "<details>"
...

Examples:
| details                            | contributions                                   | results|
| age:24, gender:male...  | employee_percent: 1 ...                  | ...        |
```